lib/docker/template/cache.rb

```diff
@@ -17,12 +17,10 @@ def context(builder, context)
         builder.repo.cache_dir.rm_rf
         $stderr.puts Simple::Ansi.yellow(format("Copying context for %s", builder.repo))
         cache_dir = builder.repo.cache_dir
-        cache_dir.parent.mkdir_p
+        cache_dir.mkdir_p
```

Comment:
About copy. I also must say that I haven't understand your copy's design, as it was not working at all for me. Maybe it was a lack of understanding about configuring it. Lots of things I had to figure out. Not all of them I did it right.